### ISSUE TITLE
Spike: CSV Errors

### ISSUE THREAD

**Issue by jillpe** [scientist-softserv/adventist-dl#526]
# Summary
https://assaydepot.slack.com/archives/C0313NJV9PE/p1692212781371739

TLDR; CSV import worked on staging, and errored out the first time on production (Ldp::NotFound). The second time it ran on production, it ran very slow and ended with a series of errors

[Importer on staging that worked](https://sdapi.s2.adventistdigitallibrary.org/importers/64?locale=en)
[Importer on production that failed](https://sdapi.b2.adventistdigitallibrary.org/importers/69?locale=en)

[Record_SPD_2023_07_15_COPY2.zip](https://github.com/scientist-softserv/adventist-dl/files/12363040/Record_SPD_2023_07_15_COPY2.zip)

# Accepted Criteria
- [ ] The problem has been identified
- [ ] Follow up ticket to fix the problem has been created

**Comment by kirkkwang** [comment]
There's a bug in Fedora that occurs when it's being overloaded with too many PUTS or POSTS that will cause these LDP errors that make the importers fail.  Generally, the solution is to initiate a reimport and if that's still not working then try a Fedora restart.

**Comment by KatharineV** [comment]
@kirkkwang Maybe we need a Fedora restart? I just tried two importers in staging/prod, and I got LDP not found in prod. 

Staging was successful: https://sdapi.s2.adventistdigitallibrary.org/importers/69?locale=en

Production failed: https://sdapi.b2.adventistdigitallibrary.org/importers/71?locale=en 

I used the same zipped files and CSV for both importers. 

I would ask if prod is failing due to the import after Spacestone, but I haven't heard that it has started (?), and I know the LDP issue was present over a month ago when we opened this ticket. What do you think?

**Comment by orangewolf** [comment]
I've fixed a few issues with the database that seem to have resolved this issue. the big was was that an index was missing on bulkrax_statuses (a very large table) causing lots of very long queries that were mucking things up. I re-ran the two importers mentioned and both ran well.

**Comment by KatharineV** [comment]
Thank you for the work you did so far! I just ran an importer on prod (since the issue was never on staging), and it ran beautifully for a minute, got stuck, and errored out with a bunch of Ldp::HttpError and Ldp::NotFound errors. Ugh! Any idea what's going on?

Here's the importer:
https://sdapi.b2.adventistdigitallibrary.org/importers/104?locale=en 

By the way, if we're on the verge of kicking off the big import post-SpaceStone, do not feel pressed to work on this ticket. I have a backlog of things I'd like to upload to the SDAPI tenant, but I know production will soon be overwhelmingly busy.
